esp/services/ws_dfu/ws_dfuXRefService.cpp

```diff
@@ -599,11 +599,25 @@ bool CWsDfuXRefEx::onDFUXRefList(IEspContext &context, IEspDFUXRefListRequest &r
 {
     try
     {
-#ifdef _CONTAINERIZED
-        IERRLOG("CONTAINERIZED(CWsDfuXRefEx::onDFUXRefList)");
-#else
         context.ensureFeatureAccess(XREF_FEATURE_URL, SecAccess_Read, ECLWATCH_DFU_XREF_ACCESS_DENIED, "WsDfuXRef::DFUXRefList: Permission denied.");
 
+#ifdef _CONTAINERIZED
+        DBGLOG("CONTAINERIZED(CWsDfuXRefEx::onDFUXRefList)");
+        Owned<IPropertyTreeIterator> planesIter = getPlanesIterator("data", nullptr);
```

Comment:
This k8s code may work for BM too.. would be good to unify if so.
BM has "data" planes automatically generated from the clusters, so it should in theory..

Reply:
Yes, the BM code now iterates through the 'clusters' using getPlanesIterator("data", nullptr). It partially worked where the Thor and Roxie clusters showed up properly and could be run, but additional 'clusters' (mythor_mirror, hthor_hthor, etc) were found that failed when trying to generate results.

I have removed the DBGLOG with a containerized message from the generalized code to avoid confusion.